creating/creating_index.adoc

```diff
@@ -491,8 +491,53 @@ include::help.adoc[]
 
 === Verifying Dockerfile (linter)
 
+The Dockerfile-lint is a rule based 'linter' for verifying Dockerfiles. The rules are used to check syntax and best practice options for things such as 'Was yum clean up evoked after a package installation?' or 'In the RUN section did the writer link commands via semicolons or double ampersands?'. The input rules are called via a set of yaml files. At the time of this writing, there are a number of templates from base to automated build configurations. 
 
+There are two iterations of the linter. 
 
+The CLI version of the project is available here:
+link:https://github.com/projectatomic/dockerfile_lint[Dockerfile_lint] 
+
+The online version of the project is available here:
+link:https://access.redhat.com/labs/linterfordockerfile/#/[Online linter]
+
+For the next section, we will assume that the linter will be installed manually via npm using the following commands: 
+
+```
+git clone https://github.com/projectatomic/dockerfile_lint/
+cd dockerfile_lint
+npm install
+```
```

Comment:
Added this and organized the information under sub headers for creating parsing rules, Using the linter and installing the linter